front-end/src/components/newsfeed-view/news-list/card-group/news-card/TwitterCard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Timestamp from './elements/Timestamp';
+import styles from './social-media-card.module.scss';
+
+const TwitterCard = ({data}) => (
+    <div className={`${styles['social-media-card']} ${styles['twitter']}`}>
+        <div className={styles["service-source"]}>
+            {data.source}
+        </div>
+        <div className={styles['content']}>
+            <div className={styles['content-source']}>
+                <img src={data.thumbnail} alt='Twitter thumbnail' />
+                <div>
+                    {data.author}
+                    <br />
+                    <span className={styles['twitter-handle']}>@{data.userhandle}</span>
```

Comment:
need to inform Shamnad that we will need this userhandle field as well, right now we have only author name and thumbnail

Reply:
Yes, Indeed. I will inform him.